app/services/book_fine_calculator.rb

```diff
@@ -0,0 +1,22 @@
+class BookFineCalculator
+  def self.including_related_checkouts(checkout)
+    new(checkout).including_related_checkouts
+  end
+
+  def initialize(checkout)
+    @checkout = checkout
+  end
+
+  def including_related_checkouts
+    sum = checkouts_for_book
```

Comment:
I don't think you need to assign the local variable `sum` here. You can return the result of the method call directly without the assignment.

Good use of inject here.